react/features/base/config/reducer.js

```diff
@@ -49,11 +49,13 @@ const INITIAL_RN_STATE = {
     disableAudioLevels: true,
 
     p2p: {
-        disableH264: false,
-        preferH264: true
-    },
-
-    remoteVideoMenu: {}
+        disableH264: false, // deprecated
+        preferH264: true, // deprecated
+        videoQuality: {
+            disabledCodec: '',
+            preferredCodec: 'H264'
```

Comment:
There is no videoQuality setting under p2p, just `preferredCodec` and `disabledCodec`. All the other settings like bitrates are applicable to both the connections and are therefore under the videoQuality settings for the whole conference. https://github.com/jitsi/jitsi-meet/blob/master/config.js#L475.

Reply:
🤦